### DISCUSSION TITLE
What YubiKey do I need to use it with KeePass?

### DISCUSSION THREAD

**Discussion by NovaViper** [keepassxreboot/keepassxc#7245] [Q&A] [answered]
Hey guys, I want to go and get a Yubikey but Im not sure which ones I need to get. There's so many different ones and I'm unsure which ones work with KeePass!

https://www.yubico.com/store/compare/

**Comment by droidmonkey** [comment, marked as answer]
Yubikey 5 only, the middle two.

**Reply by NovaViper** [reply]
Thank you!
